families/track_and_trade/server/index.js

```diff
@@ -18,16 +18,21 @@
 
 const express = require('express')
 const db = require('./db')
+const blockchain = require('./blockchain')
 const api = require('./api')
 
 const PORT = process.env.PORT || 3000
 const app = express()
 
 db.connect()
+  .then(() => {
+    blockchain.subscribe()
```

Comment:
I wonder if this should be a forked process (using the `cluster` api (https://nodejs.org/dist/latest-v6.x/docs/api/cluster.html)